### PR TITLE
First Pull Request for review

### PR DESCRIPTION
**Pull request by K0ppai**
In this project, I've accomplished the following,

- Add the required files such as index.html, style.css, linters.yml, package.json, etc.
- Check the errors of HTML & CSS language using linters
- Use professional gitflow and descriptive commits
-Customize the given Microverse template according to my project

### CODE REVIEW

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @K0ppai ,

Good job implementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:

- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please make sure Your README file looks great and professional. Kindly consider making it more descriptive by adding as many sections as possible from the [README template](https://github.com/microverseinc/readme-template) provided by Microverse and remember to customize it. 
reademe

- [ ] Please make sure your Pull Request  has a summary describing the introduced changes in more detail.  As you can see there is no description provided for this PR
prdescriptive


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @K0ppai :wave: 

Well done on implementing the changes suggested by the previous reviewer. :+1:  
You need to fix those issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ You have used the correct gitflow.
✔️ Commit message is professional

## Required Changes ♻️

- [ ] Though you have added a description in the pull request but still it is not professional. In the description you need to describe what you have done in the whole project, not in the last commit. Here I am giving you an example, Kindly consider this and fix the issue.

  - ***You can write in the pull request description:***
>     - In this project I have implemented the requirements below:
>       - Implement basic `CSS` and `HTML`.
>       - Implement `linters` and configure them with the project.
>       - Use `Microverse Readme` file and customize it.
>       - Create a `test.md` file and ignore it by using `.gitignore`.

    
image


> Kindly check the comments below.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,136 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://github.com/K0ppai/hello-microverse-project)
```
Comment (Olamarx) [inline README.md]: - [OPTIONAL] Since you have not deployed it, I strongly advise you to remove this.

**Review by Olamarx — APPROVED** [review]
Hi @K0ppai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- PR description ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**